weaviate/collections/data.py

```diff
@@ -403,7 +394,7 @@ def replace(
         self,
         uuid: UUID,
         properties: Properties,
-        references: Optional[WeaviateReferences] = None,
+        references: Optional[ReferenceInputs] = None,
```

Comment:
I don't think that we have tests that send refs as List for replace, can you add one?